### DISCUSSION TITLE
Changing the default menu icon

### DISCUSSION THREAD

**Discussion by LambLeeg** [linuxmint/discussions#140] [Ideas]
I know that it takes a couple of clicks to use an icon of your choice, but recently the Mint team started to tweak the look and feel out of the box. An icon must be simplistic, yet modern. So, I suggest the usage of that icon:

![Mint menu](https://github.com/linuxmint/discussions/assets/59935189/a269611e-8a59-4ebf-9146-7a32d32a4009)

I think that the old default icon is simply put terrible and was fine for the mint-x pack, but nowadays, it's truly orphaned. Also, that is a svg file, so that the leafs may be colored with mint-y colors, this one is an alternative default option.

**Comment by Secret-chest** [comment]
This is not simplistic at all! And it is not even the logo, it becomes even more cluttered.

I, for one, would like to see something even simpler:

![image](https://github.com/linuxmint/discussions/assets/74449186/fdf34167-2287-4517-902b-39294a14209c)
